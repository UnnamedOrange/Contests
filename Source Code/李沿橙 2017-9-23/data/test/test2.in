5
8
5 13 19 37 51 46 24 14 
8
1 2 3 4 5 6 7 50 
9
335 383 492 777 886 915 793 649 386 
10
1 2 3 5 9 10 11 12 13 14 
12
1 990 991 992 993 994 995 996 997 998 999 1000 
