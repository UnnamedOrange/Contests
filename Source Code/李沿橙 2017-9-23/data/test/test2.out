5 13 14 19 37 51 46 24 
1 2 3 4 5 6 50 7 
335 383 386 649 793 886 915 777 492 
1 2 3 9 10 12 13 14 11 5 
1 990 992 993 994 995 996 997 998 999 1000 991 
